tox.ini

```diff
@@ -303,3 +271,151 @@ commands = python -m autonomy.cli analyse abci check-app-specs --check-all
 skipsdist = True
 usedevelop = True
 commands = python -m autonomy.cli analyse abci check-handlers --common abci,http,contract_api,ledger_api,signing --skip abstract_abci,counter,counter_client,hello_world_abci
+
+
+[testenv_multi_ubuntu]
+basepython = python3
+whitelist_externals = /bin/sh
+passenv = *
+extras = all
+deps =
+    requests==2.26.0
```

Comment:
A lot of libs are shared between all three envs. Please create a base env with the shared libs and then specify the ones with different versions here